core/lib/lantern-trace-saver.js

```diff
@@ -31,7 +31,11 @@ function convertNodeTimingsToTrace(nodeTimings) {
     } else {
       // Ignore data URIs as they don't really add much value
       if (/^data/.test(node.record.url)) continue;
-      traceEvents.push(...createFakeNetworkEvents(node.record, timing));
+      if (requestId === 1) {
+        traceEvents.push(createWillSendRequestEvent(requestId, node.record, timing));
```

Comment:
> you could do this conditional within `createFakeNetworkEvents` instead.

yeah, I was trying to keep `createFakeNetworkEvents` ignorant of the `requestId`, but it's already pretty hacky, so that's fine. I suspect this needs to be reworked for redirects and timespans anyway.